.eslintrc.js

```diff
@@ -32,6 +31,16 @@ module.exports = {
     '@next/next/no-html-link-for-pages': ['off'],
     '@next/next/no-img-element': ['off'],
     'linebreak-style': ['error', os.platform() === 'win32' ? 'windows' : 'unix'],
+    'import/extensions': [
+      'error',
+      'ignorePackages',
+      {
+        js: 'never',
+        jsx: 'never',
+        ts: 'never',
+        tsx: 'never',
+      },
+    ],
```

Comment:
this change is technically out of scope for this PR - we would want to make a follow up issue to address this specifically - please revert the changes in this file

Reply:
Done